pkg/flow/import_test.go

```diff
@@ -0,0 +1,769 @@
+package flow_test
```

Comment:
This file is very big. You might want to consider using testdata and [txttar](https://pkg.go.dev/golang.org/x/tools/txtar) as a way to better organize this. For other testing tips, I'd recommend watching the talk [Go Testing By Example](https://research.swtch.com/testing).